src/GeositeFramework/js/SubRegion.js

```diff
@@ -58,17 +75,44 @@
     }
 
     function addSubRegionsToMap(subregions, layer) {
-
         _.each(subregions.areas, function(subregion) {
             var geom = new esri.geometry.Polygon(subregion.shape);
             var symbol = new esri.symbol.SimpleFillSymbol();
             symbol.setColor(new dojo.Color(subregion.color || subregions.color));
 
-            var graphic = new esri.Graphic(geom, symbol);
-            graphic.attributes = subregion;
+            var infoTemplate = new esri.InfoTemplate();
+            infoTemplate.setContent("${display}");
 
-            layer.add(graphic);
+            var graphic = new esri.Graphic(geom, symbol, subregion, infoTemplate);
 
+            layer.add(graphic);
         });
     }
-}(Geosite));
+
+    function createTooltip(map, Popup) {
+        var mapContainerId = $(map.__container).parent().parent().attr('id'),
+            el = $('#' + mapContainerId).find('.subregion-tooltip')[0],
+            tooltip = new Popup({
+                map: map,
+                anchor: 'top'
+            }, el);
+
+        return tooltip;
+    }
+
+    function showTooltip(e, tooltip) {
+        var extent = e.graphic._extent,
+            height = extent.getHeight();
+
+        tooltip.setContent(e.graphic.getContent());
+        // The Y coordinate of the tooltip should be place above the
+        // graphic, which is a function of the graphic's height.
+        // height * 30 seems about right.
+        tooltip.offsetY = height * 30;
```

Comment:
`extent` is bounding box in the ESRI world. I did find another way to access it besides the private property, though.